src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -375,6 +375,15 @@ public function getCacheDir()
         return $this->rootDir.'/cache/'.$this->environment;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getTmpDir()
+    {
+        // Returns $this->getCacheDir() for backward compatibility
+        return $this->getCacheDir();
```

Comment:
Here is the core of the PR :)

For backward compatibility, I return the current cache directory.

In future versions, this could be for example `var/tmp`. In the meantime, users can override this method to set it to a different path, for example `/tmp`.